tests/test_multipart.py

```diff
@@ -7,10 +7,10 @@
 import pytest
 
 import httpx
-from httpx.config import CertTypes, TimeoutTypes, VerifyTypes
-from httpx.content_streams import encode
-from httpx.dispatch.base import AsyncDispatcher
-from httpx.utils import format_form_param
+from httpx._config import CertTypes, TimeoutTypes, VerifyTypes
```

Comment:
Incidentally, it looks like we've erronously left the signature as `send(request, verify, cert, timeout)` in a bunch of these and other test cases, rather than switching to `send(request, timeout)` on the dispatch interface.

We're missing those because we're not type checking our test cases.

Not related to this PR, so doesn't neccessarily need to change here, but this helps highlight it, since we're pulling in more private API that we might otherwise expect.